package.json

```diff
@@ -27,6 +27,7 @@
   ],
   "scripts": {
     "preinstall": "node -e \"process.exit(process.env.npm_execpath.includes('yarn') ? 0 : 1)\"",
+    "hops": "yarn workspace hops-template-react hops",
```

Comment:
For me the hops shortcut has the purpose to be not required to know/remember how the demo app is named for each  repository.

Reply:
But we have more than one demo app. We have three distinct templates here.
And by doing a `yarn run` and then `ctrl+c` you can see all scripts that are available ;)

Reply:
Sure, but I would like to have one demo app to be the default and we use hops-template already for `yarn start` as the default.
The nice thing about `yarn hops` is that it works the same in a workspace project as in a normal one. And instead of redirecting `yarn build` `yarn start` `yarn fetch-schema` we only have one redirect and it works for all Hops commands.